blocks/library/paragraph/test/__snapshots__/index.js.snap

```diff
@@ -2,30 +2,32 @@
 
 exports[`core/paragraph block edit matches snapshot 1`] = `
 <div>
-  <div>
-    <div
-      class="components-autocomplete"
-    >
-      <div
-        class="blocks-rich-text"
-      >
-        <p
-          aria-autocomplete="list"
-          aria-expanded="false"
-          aria-label="Add text or type / to add content"
-          aria-multiline="false"
-          class="wp-block-paragraph blocks-rich-text__tinymce"
-          contenteditable="true"
-          data-is-placeholder-visible="true"
-          role="textbox"
-        />
-        <p
-          class="blocks-rich-text__tinymce wp-block-paragraph"
+  <div
```

Comment:
So it looks like this issue with the obsolete `div` elements isn't something new. Let's leave it as it is and circle back later. We can investigate if React 16.3 has something to offer to mitigate it. https://reactjs.org/blog/2018/03/29/react-v-16-3.html - introduces `createRef` and `forwardRef` - they might be what we need in this case. We still need to expose those methods in a separate PR.